File: candidate-client/src/app/models/ReportSessionDTO.ts
export class ReportSessionDTO {

	sessionStart: Date;
	sessionEnd: Date;
	durationInHours: Number;
	hourTypeId: Number;
	hourTypeName: String;
	ratio: Number;
	value: Number;

	constructor(obj: any) {

		Object.assign(this, obj);

		this.sessionStart = new Date(obj.sessionStart);
		this.sessionEnd = new Date(obj.sessionEnd);

	}

}

File: candidate-api/src/main/java/com/nebu/candidate/dto/report/ReportSessionDTO.java
package com.nebu.candidate.dto.report;

import lombok.*;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@EqualsAndHashCode
public class ReportSessionDTO {

	private String sessionStart;

	private String sessionEnd;

	private double durationInHours;

	private long hourTypeId;

	private String hourTypeName;

	private double ratio;

	private double value;

}
